.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
this is ok, but please try and be more careful with your commits, because even though you have deleted this, it stays in the history.   Only add files to the commit that you want to commit.  You can update ``.gitignore`` if you need to.